controllers/get-article.js

```diff
@@ -0,0 +1,15 @@
+
+const fetchArticle = require(`../modules/fetch-article`);
+
+module.exports = function getArticle(req, res, next) {
+	fetchArticle(req.params.article_id)
+		.then(article => {
+			if (!article) {
+				res.status(404).send('error: path or content not found');
+			}
```

Comment:
You should do this in the model with a promise reject rather than sending back a different type of response here. 

If you promise.reject from the model then you can catch the err below and pass it to an error handler.

That way you have one place that will deal with all of these types of errors instead of repeating yourself in controllers!